devops/automation/azure-pipelines.yml

```diff
@@ -116,29 +116,29 @@ jobs:
     displayName: 'Clean workspace'
     workingDirectory: $(Build.SourcesDirectory)
 
-  - script: |
-      set -e
-      set -x
-      $(Build.SourcesDirectory)/binding-tools-for-swift/devops/automation/build-swift.sh --publish
-      mkdir -p $(System.DefaultWorkingDirectory)/toolchain
-      if test -f $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip; then
-        cp -c $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip $(System.DefaultWorkingDirectory)/toolchain
-        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]1"
-      else
-        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]0"
-      fi
-    displayName: 'Build Swift'
-    name: BuildSwift
+#  - script: |
+#      set -e
+#      set -x
+#      $(Build.SourcesDirectory)/binding-tools-for-swift/devops/automation/build-swift.sh --publish
+#      mkdir -p $(System.DefaultWorkingDirectory)/toolchain
+#      if test -f $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip; then
+#        cp -c $(System.DefaultWorkingDirectory)/binding-tools-for-swift/Pack-Man/SwiftToolchain-*.zip $(System.DefaultWorkingDirectory)/toolchain
+#        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]1"
+#      else
+#        echo "##vso[task.setvariable variable=uploadtoolchain;isOutput=true]0"
+#      fi
+#    displayName: 'Build Swift'
+#    name: BuildSwift
 
-  # This will upload the updated toolchain zip (if any) to Azure Devops
-  # A later job further down will then publish the toolchain to Azure Blob Storage.
-  # This is a two-step process because the task that can publish to Azure Blob Storage only works on Windows :/
-  - task: PublishBuildArtifacts@1
-    displayName: 'Upload Swift Toolchain'
-    inputs:
-      PathtoPublish: $(System.DefaultWorkingDirectory)/toolchain
-      ArtifactName: 'toolchain'
-      publishLocation: 'Container'
+#  # This will upload the updated toolchain zip (if any) to Azure Devops
+#  # A later job further down will then publish the toolchain to Azure Blob Storage.
+#  # This is a two-step process because the task that can publish to Azure Blob Storage only works on Windows :/
+#  - task: PublishBuildArtifacts@1
+#    displayName: 'Upload Swift Toolchain'
+#    inputs:
+#      PathtoPublish: $(System.DefaultWorkingDirectory)/toolchain
+#      ArtifactName: 'toolchain'
+#      publishLocation: 'Container'
```

Comment:
Same:
```suggestion
  # This will upload the updated toolchain zip (if any) to Azure Devops
  # A later job further down will then publish the toolchain to Azure Blob Storage.
  # This is a two-step process because the task that can publish to Azure Blob Storage only works on Windows :/
  - task: PublishBuildArtifacts@1
    displayName: 'Upload Swift Toolchain'
    enabled: false
    inputs:
      PathtoPublish: $(System.DefaultWorkingDirectory)/toolchain
      ArtifactName: 'toolchain'
      publishLocation: 'Container'
```